### PR TITLE
Added wallet address param

### PR DESCRIPTION
**Pull request by Polybius93**

### CODE REVIEW

File: src/background/PopupHandler/index.tsx [hunk 1]
```diff
@@ -24,6 +24,11 @@ export const PopupHandler: FC = () => {
   const [displayError, setDisplayError] = useState(true);
   const [processRequested, setProcessRequested] = useState(false);
   const snackbar = useSnackbar();
+  const [wallet, setWallet] = useState('')
+
+  useEffect(() => {
+    console.log(wallet)
+  })
```
Comment (sosaucily) [inline src/background/PopupHandler/index.tsx:31]: Logs!

File: src/background/PopupHandler/index.tsx [hunk 2]
```diff
@@ -35,7 +40,7 @@ export const PopupHandler: FC = () => {
 
   useEffect(() => {
     if (processRequested && success) {
-      navigate(`/contractdisplay/${curContractId}`);
+      navigate(`/contractdisplay/${curContractId}/${wallet}`);
```
Comment (sosaucily) [inline src/background/PopupHandler/index.tsx:43]: Usually we use two 'r's as in current. So currContractId

**Review by sosaucily — APPROVED** [review]
tiny fixes

File: src/background/PopupHandler/index.tsx (hunk 1 above)
Comment (Polybius93) [reply src/background/PopupHandler/index.tsx:31]: Sorry about that!

File: src/background/PopupHandler/index.tsx (hunk 2 above)
Comment (Polybius93) [reply src/background/PopupHandler/index.tsx:43]: All right!

**Pull request merged**